code/game/jobs/job/support.dm

```diff
@@ -174,6 +174,8 @@
 	singlemutcheck(H, GLOB.soberblock, MUTCHK_FORCED)
 	H.dna.default_blocks.Add(GLOB.soberblock)
 	H.check_mutations = 1
+	ADD_TRAIT(H, TRAIT_TABLE_LEAP, ROUNDSTART_TRAIT)
+	ADD_TRAIT(H, TRAIT_SLEIGHT_OF_HAND, ROUNDSTART_TRAIT)
```

Comment:
Add it on the mind, like table leap, and remove the second addition of the trait.